### ISSUE TITLE
Recruit SIP Working Group Members

### ISSUE THREAD

**Issue by jennymith** [stacksgov/pm#142]
**Reference:** https://forum.stacks.org/t/respond-by-may-6th-call-for-sip-working-groups/11981

- [x] Discuss process for onboarding working group members with Stacksgov
- [x] Share SIP working group opportunity widely
- [ ] Invite potential members to informal interview or next governance meeting
- [ ] Finalize working group members for each consideration by mid-May

**Comment by whoabuddy** [comment]
@whoabuddy to finalize Governance CAB as an example for others moving forward (by mid-August).

**Comment by whoabuddy** [comment]
Linking some examples of how the Governance CAB was formed, happy to help any future CABs get acquainted with the process!

- [adding the CAB members info](https://github.com/stacksgov/sips/pull/33)
- [first meeting minutes](https://github.com/stacksgov/sips/pull/46)

**Comment by whoabuddy** [comment]
During #179 @hero-gamer shared that the experience of having to submit the CAB request as a PR on GitHub presented a bit of friction as a non-GitHub user.

Reference: stacksgov/sips#65

- if a user does not have an account, they need to set one up
- if they are not familiar with source control or development work
  - there are a lot of terms that require additional research
  - the process of forking and submitting a PR has a lot of steps

Shout out to @hozzjss for helping out!

This could be addressed by writing up a standards and procedures document that walks through the flow for submitting yourself to a CAB, submitting meeting minutes for a CAB, or similar.

This can also be addressed by reaching out to someone already involved with the StacksGov GitHub, I'm willing to help anyone get setup if they want to be a part of one.

**Comment by Jolie-Ze** [comment]
I'll just add some notes here from Jasper and my conversation this week regarding recruitment and education:

**Creating a CAB onboarding ramp/journey**
Establishing the details of the CAB involvement in layman's terms:
- How to get involved in governance
- Decision making around SIPs every 3 months
- Monday Morning 8am PT weekly blockchain engineering meetings to stay up to date
- Discussions feature on Stacks blockchain github repo
- [ ] Look into a tool to integrate GH discussion and pages
- Confront obstacles in language and destination
- [ ] Better links to more information ? For reference: https://stacks101.com/

This will likely end up as an action item but placing it here for now 🙏🏼

**Comment by Hero-Gamer** [comment]
Hi all, 

Just to give an update to everyone, about the work currently underway.
(I'm not officially a SIP Resident yet (hopefully soon to be), however, I'm getting work started already anyways.)

- I'm reaching out to many individuals in the community and major independent entities to confirm their interest in contributing to CABs, based on their domain of expertise and other factors. 
- **I shall aim to complete and confirm all CAB members within a month or so.** 
- On a slightly separate but related note, search for SIP Editors also underway, this will take little longer but also getting positive feedback so far. I should have minimum of 10 core SIP Editors, and more. 

**Will keep everyone updated.** :) :rocket:
_In the meantime, if you have any suggestion of anyone who might be a good fit for CAB or Editor, do reach out to me! lfg_

**Comment by HaroldDavis3** [comment]
> Hi all,
> 
> Just to give an update to everyone, about the work currently underway. (I'm not officially a SIP Resident yet (hopefully soon to be), however, I'm getting work started already anyways.)
> 
> * I'm reaching out to many individuals in the community and major independent entities to confirm their interest in contributing to CABs, based on their domain of expertise and other factors.
> * **I shall aim to complete and confirm all CAB members within a month or so.**
> * On a slightly separate but related note, search for SIP Editors also underway, this will take little longer but also getting positive feedback so far. I should have minimum of 10 core SIP Editors, and more.
> 
> **Will keep everyone updated.** :) 🚀 _In the meantime, if you have any suggestion of anyone who might be a good fit for CAB or Editor, do reach out to me! lfg_

Let us know of any updates here mane :) and congrats on the residency again!

**Comment by Hero-Gamer** [comment]
Hi all, 

Some updates to keep everyone in the loop:

**1. We have 22 individual CAB members, 5 CABs.** See list or all members below. Thankful we have great community who are willing to contribute towards this cause

**2. Activation process began**: currently WIP. I've let people know some tasks need to be completed, e.g. put names bio on Github, agree on chairperson, agree parameter for reviewing SIP, announcing their panel to the wider community etc.
Anyone can see the blog post about how to become CAB member, this will contain all info required: https://app.sigle.io/herogamer.id.stx/sz6Sy0cV--XsWDBYQQ_Dq 
Message to ask members to start activation process here: https://discord.com/channels/621759717756370964/1001471092302544936/1016803211358781461

**3. CAB Q&A with Jude:** Tomorrow 9th September Friday, a meeting of with all CAB members and Jude will be available on SIP calls to answer any questions they may have before full activation of boards.
<img src="https://user-images.githubusercontent.com/92864860/189215663-58cd4264-3bee-4d41-8330-bc2bbdbac25e.png" width="400px" />

⚡ Governance CAB - 5 members
Jason, Harold, Orlando, Juliet, Zero.btc 
⚡ Technical CAB - 8 members
Aaron, Brice, Jesse W., Jamil, Terje, Dan, Thomas (Aulneau), Mike
⚡ Economics CAB - 3 members
Matty, Xan, Chiente
⚡ Diversity CAB - 4 members
Shannon, HeroGamer, Brett (Eliherf), Nick
⚡ Ethics CAB - 3 members
Juliet, James (hodlstx.btc), Sam (SJC)

🐘 The Steering Committee (SC) line up are:
Jude, Gina

On a related note:
**4. Steering Committee meeting:** the last Friday of end of each month, there will be a Steering Committee update, which we currently asking CAB members to attend. Will see how it goes.

**TLDR: Still some work to do. I will count them fully "recruited" once all members have put their name into Github and announced their Board with the community via Stacks Forum & Github.**

**Comment by Hero-Gamer** [comment]
Hi all & @jennymith , would like to update everyone that....
* **I have successfully recruited and activated 3 Consideration Advisory Boards (CABs) as part of my Residency goals now!!!**
* **Made up of 17 excellent experts from all across the ecosystem. 9 in Tech; 5 in Gov; 3 in Econ**

**Twitter** **announcement** 
from Stacks Foundation: https://twitter.com/stacksorg/status/1575228593432514560?s=21&t=1wbSVSV-23nQ5FxiFsUMCA
from Hero: https://twitter.com/herogamer21btc/status/1575153709079703552?s=21&t=1wbSVSV-23nQ5FxiFsUMCA
**Stacks Forum** **announcement** 
Technical CAB: https://forum.stacks.org/t/activation-of-technical-consideration-advisory-board-cab/13761
Governance CAB: https://forum.stacks.org/t/activation-of-governance-consideration-advisory-board-cab/13763
Economics CAB: https://forum.stacks.org/t/activation-of-economics-consideration-advisory-board-cab/13762

**Chairperon for each CABs**
Technical CAB: Brice
Governance CAB: Jason
Economics CAB: MattySTX

**You can read all the CABs' considerations here & all members' bio:**
Technical: https://github.com/stacksgov/sips/blob/main/considerations/technical.md
Governance: https://github.com/stacksgov/sips/blob/main/considerations/governance.md
Economics: https://github.com/stacksgov/sips/blob/main/considerations/economics.md

<img src="https://user-images.githubusercontent.com/92864860/192899125-4c52b231-6e10-4b26-837e-9f2fc015db91.JPG" width="300px" />
<img src="https://user-images.githubusercontent.com/92864860/192899136-3d20df8e-55a2-4055-8f15-4741c65ef71a.JPG" width="300px" />
<img src="https://user-images.githubusercontent.com/92864860/192899143-da81c73e-f99a-4b03-8e2a-db63b2a7323a.JPG" width="300px" />

* This is not the end of the process, this is only the beginning. More recruitment and additions will be needed, particularly on CAB such as Economics, having 3 people only is not sustainable. 

* Two other groups **Diversity** and **Ethics** have had more extensive discussion as to what form their group takes place, which may not end up being CABs. They require further discussion and sense-making to flesh out how best they exist and communicate in the SIP process.

* This should solve the and mark the original post: https://forum.stacks.org/t/respond-by-may-6th-call-for-sip-working-groups/11981 & this Github issue#142
* So I'd like to propose we close this issue and mark it complete. What do you guys think? @jennymith 

* Discussion on Diversity and Ethics with their members are still currently going on. I can fire up new issues to ensure they are being worked on if need be.

**Comment by whoabuddy** [comment]
Thanks @Hero-Gamer for all the work here, great job!

Closing this out as the main task is complete - future CAB work can use the `mtg-discuss` or `mtg-action` to discuss items relevant to new formations.
